refactor/imports.go

```diff
@@ -38,11 +38,12 @@ func deleteUnusedImports(s *Snapshot, p *Package, text []byte) []byte {
 
 	match := func(name, pkg string) bool {
 		if name == "" {
-			p1 := s.pkgGraph.byPath(pkg)
+			var p1 *Package
+			if path, ok := p.ImportMap[pkg]; ok {
+				p1 = s.pkgGraph.pkgByPath[path]
```

Comment:
Use `s.pkgGraph.byPath(path)` here? It's the same thing, but more consistent with Import.